server.js

```diff
@@ -153,7 +156,7 @@ function Server (opts) {
       if (req.method === 'GET' && (req.url === '/stats' || req.url === '/stats.json')) {
         infoHashes.forEach(function (infoHash) {
           var peers = self.torrents[infoHash].peers
-          var keys = Object.keys(peers)
+          var keys = Object.keys(peers.cache)
```

Comment:
@yciabaud Line 158 should be the array of peers, not the LRU instance. Hence this comment https://github.com/feross/bittorrent-tracker/pull/160/files/7f98203d5ec63f395e2794700a10046c54dcf8f1#r66526332